docs/templating_funcs.md

```diff
@@ -133,3 +133,9 @@ The `expandSecretRefs` function takes an object as the argument and expands ever
 ```yaml
 {{ $expandSecretRefs :=  $value | expandSecretRefs }}
 ```
+
+#### `include`
+The 'include' function allows including and rendering nested templates. The function returns the created template or an error if any occurred. It will load functions from `_*.tpl` files in the directory specified by the `-f` option.
```

Comment:
```suggestion
The 'include' function allows including and rendering nested templates. The function returns the created template or an error if any occurred. It will load functions from `_*.tpl` files in the directory where the helmfile.yaml is located.

For nested helmfile.yaml files, it will load `_*.tpl` files ni the directory where each nested helmfile.yaml is located.
```

As per https://github.com/helmfile/helmfile/pull/1187/files#r1412908641